consultation_analyser/consultations/models.py

```diff
@@ -477,3 +478,68 @@ class EvidenceRichMapping(UUIDPrimaryKeyModel, TimeStampedModel):
 
     class Meta(UUIDPrimaryKeyModel.Meta, TimeStampedModel.Meta):
         pass
+
+
+class Response(models.Model):
+    id: models.UUIDField = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+    consultation: models.ForeignKey = models.ForeignKey(
+        "Consultation", on_delete=models.CASCADE, related_name="responses"
+    )
+    respondent: models.ForeignKey = models.ForeignKey(
+        "Respondent", on_delete=models.CASCADE, related_name="responses"
+    )
+    question: models.ForeignKey = models.ForeignKey(
+        "Question",
+        on_delete=models.CASCADE,
+        related_name="responses",
+        null=True,
+        blank=True,
+    )
+
+    question_text_ft = models.TextField()
+    question_text_mc = models.TextField(blank=True, null=True)
```

Comment:
Do we need to store the question_text fields on the Response, given that they will be in the question? From a FE point of view, I can't imagine we'll need to return this for each response in a particular set of responses in the 'list responses by question' case. If we have a 'list responses by respondent' case in the future, I can't imagine we'll have so many questions that makes the extra query to the foreign question object a massive time-sink